lib/rdoc/code_object.rb

```diff
@@ -424,4 +427,19 @@ def suppressed?
     @suppressed
   end
 
+  ##
+  # Autolink in cross reference?
+
+  def autolink?
+    autolink = @metadata["autolink"]
+    if autolink
+      RDoc::Options.boolean(autolink, "autolink")
+    else
+      true
+    end
```

Comment:
Can we simplify this?

```suggestion
    RDoc::Options.boolean(@metadata["autolink"] || true, "autolink")
```

Reply:
`fetch` may be better?

```suggestion
    RDoc::Options.boolean(@metadata.fetch("autolink", true), "autolink")
```

Reply:
Yes.